source/reference/minio-mc/mc-replicate-diff.rst

```diff
@@ -0,0 +1,142 @@
+.. _minio-mc-replicate-diff:
+
+=====================
+``mc replicate diff``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc replicate diff
+
+Description
+-----------
+
+.. start-mc-replicate-diff-desc
+
+The :mc:`mc replicate diff` shows a list of ``PUT`` or ``DELETE`` actions to objects and object versions that have not replicated to a remote bucket.
+
+.. end-mc-replicate-diff-desc
+
+You can specify a configured remote target ARN to shows ``PUT`` and ``DELETE`` actions that have not yet replicated to the specific remote target.
+
+To find the replication status of objects for a particular remote target, you must have the ARN of the remote target.
+You can use :ref:`retrieve the remote targets configured for a bucket <minio-retrieve-remote-bucket-targets>`.
+
+Syntax
+------
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command shows unreplicated ``PUT`` or ``DELETE`` actions to objects and object versions to the remote target at ARN ``arn:minio:replication::3bb8c736-4014-42c5-b3cb-d64e3ebaa75e:notes`` for the ``teamorange/projects`` prefix in the ``notes`` bucket for the ``myminio`` ALIAS:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc replicate diff myminio/notes/teamorange/projects --arn arn:minio:replication::3bb8c736-4014-42c5-b3cb-d64e3ebaa75e:notes
+
+      If any unreplicated actions exist, the command outputs something similar to the following:
+
+      .. code-block:: shell
+
+         [0001-01-01 00:00:00 UTC] [2022-10-06 17:18:59 UTC]          478efe49-aa9d-46ab-8268-45b70cc4c341 PUT agenda.docx
+         [0001-01-01 00:00:00 UTC] [2022-10-06 17:18:15 UTC]          b283bf43-319f-455a-a779-3c2e669fad88 PUT budget-meeting.docx
+
```

Comment:
Maybe here we can note that `PUT` corresponds to a new object, `DEL` corresponds to a deleted object.

Reply:
Added.